lbz/events/broker.py

```diff
@@ -40,3 +44,55 @@ def _get_handlers(self) -> List[Callable]:
             return self.mapper[self.event.type]
         except KeyError as err:
             raise NotImplementedError(f"No handlers implemented for {self.event.type}") from err
+
+
+class EventBroker(BaseEventBroker):
+    def __init__(
+        self,
+        mapper: Mapping[str, List[Callable[[Event], None]]],
+        event: dict,
+        context: LambdaContext,
+        *,
+        type_key: str = EVENT_BRIDGE_DEFAULT_TYPE_KEY,
+        data_key: str = EVENT_BRIDGE_DEFAULT_DATA_KEY,
```

Comment:
Let's remove these arguments to make `EventBroker` our standard class as it was agreed in the past 😉 

Of course, this won't be backward compatible but if really someone is using these arguments, they will simply add a `Base` prefix before the name of the class 😉